app/models/questionable/question.rb

```diff
@@ -20,24 +20,25 @@ def #{type}?
     end
 
     def answers_for_user(user)
-      answers = self.answers.where(user_id: user.id)
-      answers.any? ? answers : [self.answers.build(user_id: user.id)]
+      answers_for_user = answers.where(user_id: user.id)
+      return answers_for_user if answers_for_user.any?
```

Comment:
For the sake of consistency this should probably call `to_a` instead of returning an AR relationship.